src/style.css

```
@@ -0,0 +1,89 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  min-height: 300px;
+  min-width: 300px;
+  flex-basis: 100%;
+  font-size: 100px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: white;
+}
+
+.block-1 {
+  background-color: red;
+}
+
+.block-2 {
```

```suggestion
..block:nth-child(2) {
```
Якщо ми маємо декілька однотипних елементів, то простіше і лаконічніше використовувати саме псевдоклас :nth-child, а не створювати безліч класів. тут у нас 6 блоків, тому це не сильно затратно. А якби було 60 чи 600? 600 окремих класів це не раціонально

---

Add fix.